### PR TITLE
[DOC] Document manual rolling updates of Connect and MM2 clusters

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Documentation

### Description

This PR does the documentation for the PR #9153 / Issue #9142. It extends the existing docs for the manual rolling updates with mentions of Connect and MM2. It uses separate PR for the docs for easier review.

### Checklist

- [x] Update documentation

### CODE REVIEW

File: documentation/assemblies/managing/assembly-rolling-updates.adoc [hunk 1]
```diff
@@ -3,9 +3,9 @@
 // using/assembly-management-tasks.adoc
 
 [id='assembly-rolling-updates-{context}']
-= Starting rolling updates of Kafka and ZooKeeper clusters using annotations
+= Starting rolling updates of Kafka, Connect, MirrorMaker 2, and ZooKeeper clusters using annotations
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/assemblies/managing/assembly-rolling-updates.adoc:6]: Trying to shrink the title a bit

File: documentation/modules/managing/proc-manual-rolling-update-pods.adoc [hunk 2]
```diff
@@ -5,12 +5,12 @@
 [id='proc-manual-rolling-update-pods-{context}']
 = Performing a rolling update using a pod annotation
 
-This procedure describes how to manually trigger a rolling update of an existing Kafka cluster or ZooKeeper cluster using a Kubernetes `Pod` annotation.
+This procedure describes how to manually trigger a rolling update of an existing Kafka, Connect, MirrorMaker2, or ZooKeeper clusters using a Kubernetes `Pod` annotation.
```
Comment (PaulRMellor) [inline documentation/modules/managing/proc-manual-rolling-update-pods.adoc:8]:
```suggestion
This procedure describes how to manually trigger a rolling update of existing Kafka, Kafka Connect, MirrorMaker 2, or ZooKeeper clusters using a Kubernetes `Pod` annotation.
```

File: documentation/modules/managing/proc-manual-rolling-update-pods.adoc [hunk 3]
```diff
@@ -5,12 +5,12 @@
...
 When multiple pods are annotated, consecutive rolling updates are performed within the same reconciliation run.
 
 .Prerequisites
 
-To perform a manual rolling update, you need a running Cluster Operator and Kafka cluster.
+To perform a manual rolling update, you need a running Cluster Operator and Kafka, Connect, or MirrorMaker2 cluster.
```
Comment (PaulRMellor) [inline documentation/modules/managing/proc-manual-rolling-update-pods.adoc:13]:
```suggestion
To perform a manual rolling update, you need a running Cluster Operator. 
The cluster for the component you are updating, whether it's Kafka, Kafka Connect, or MirrorMaker 2, must also be running.
```

Comment (PaulRMellor) [reply documentation/modules/managing/proc-manual-rolling-update-pods.adoc:13]: we don't mention ZooKeeper here?

File: documentation/modules/managing/proc-manual-rolling-update-strimzipodset.adoc [hunk 4]
```diff
@@ -6,18 +6,20 @@
...
 
 [role="_abstract"]
-This procedure describes how to trigger a rolling update of a Kafka cluster or ZooKeeper cluster.
+This procedure describes how to trigger a rolling update of Kafka, Connect, MirrorMaker2, or ZooKeeper clusters.
 To trigger the update, you add an annotation to the `StrimziPodSet` that manages the pods running on the cluster.
 
 .Prerequisites
 
-To perform a manual rolling update, you need a running Cluster Operator and Kafka cluster.
+To perform a manual rolling update, you need a running Cluster Operator and Kafka, Connect, or MirrorMaker2 cluster.
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/modules/managing/proc-manual-rolling-update-strimzipodset.adoc:14]: No ZooKeeper here?

File: documentation/modules/managing/proc-manual-rolling-update-strimzipodset.adoc [hunk 5]
```diff
@@ -6,18 +6,20 @@
...
 
 .Procedure
 
-. Find the name of the resource that controls the Kafka or ZooKeeper pods you want to manually update.
+. Find the name of the resource that controls the pods you want to manually update.
 +
 For example, if your Kafka cluster is named _my-cluster_, the corresponding names are _my-cluster-kafka_ and _my-cluster-zookeeper_.
+For Connect cluster named _my-connect_, the corresponding name is _my-connect-connect_.
+And for MirrorMaker 2 cluster named _my-mm2_, the corresponding name is _my-mm2-mirrormaker2_.
```
Comment (PaulRMellor) [inline documentation/modules/managing/proc-manual-rolling-update-strimzipodset.adoc:22]:
```suggestion
And for a MirrorMaker 2 cluster named _my-mm2-cluster_, the corresponding name is _my-mm2-cluster-mirrormaker2_.
```

**Review by PaulRMellor — APPROVED** [review]
Looks good. Few minor suggestions.
I suggested the name changes of the pods and resources so they weren't mistaken for typos. ;-)

File: documentation/assemblies/managing/assembly-rolling-updates.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (scholzj) [reply documentation/assemblies/managing/assembly-rolling-updates.adoc:6]: @PaulRMellor "related clusters" seems weird. What if we use `Starting rolling updates of Kafka and other operands using annotations`?

[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/assemblies/managing/assembly-rolling-updates.adoc:6]: That's good for me

**Pull request merged**